resources/styles/components/student-dashboard/progress-guide.less

```diff
@@ -8,26 +8,42 @@
     h1,h2,h3 { margin: 0; text-align: left; }
     .panel-title {
       #fonts > .sans(2.5rem, 5rem);
+      font-weight: 900;
       border-bottom: thin dotted @tutor-neutral-light;
     }
-    .current { #fonts > .sans(1.6rem, 5rem); }
+    .current {
+      #fonts > .sans(1.6rem, 5rem);
+      font-weight: 600;
+    }
     .btn-group {
+      width: 100%;
       // override material design
       .btn:not(.btn-link):not(.btn-flat):not(.btn-fab) { margin-right: 0; }
-      .stronger i {
-        float: left;
-        &:after {
-          .fa-icon();
-          padding-right:8px;
-          content: @fa-var-arrow-up;
-          color: @tutor-success;
+      .stronger {
+        width: 50%;
+        span { float: right; }
+
+          i {
+          float: left;
+          &:after {
+            .fa-icon();
+            padding-right:8px;
+            content: @fa-var-arrow-up;
+            color: @tutor-success;
+          }
         }
       }
-      .weaker i:after {
-        .fa-icon();
-        padding-left:8px;
-        content: @fa-var-arrow-down;
-        color: @tutor-danger;
+      .weaker {
+        width: 50%;
+        border-top-left-radius: 0 !important;
+        border-bottom-left-radius: 0 !important;
+
```

Comment:
Are you sure the buttons have a border-radius set?  I don't see any rules specifying that and they appear to be in the "flat" style to me.

If they do have a border-radius, the `!important` clause could be dropped if you change the rule to be as specific as whatever applies it.  For instance, here's a case where I had to get pretty specific in order to override some material-design styles: https://github.com/openstax/tutor-js/blob/master/resources/styles/components/student-dashboard/progress-guide.less#L16